applications/API/endpoints.py

```
@@ -0,0 +1,780 @@
+        201 Created: Insult object, including generated `reference_id`.
+        400 Bad Request: Invalid input.
+        401 Unauthorized: Missing/invalid token.
+
+    Notes
+        The endpoint records the submitting user as `added_by`.
+    """
+    serializer_class = CreateInsultSerializer
+    permission_classes = [IsAuthenticated]
+    authentication_classes = [TokenAuthentication]
+
+    def post(self, request, *args, **kwargs):
```

Pylint (3.3.7)

[warning] Useless parent or super() delegation in method 'post'

(W0246)